src/fixed_width_ints.rs

```diff
@@ -20,7 +20,7 @@ pub type uint32_t = u32;
 pub type uint64_t = u64;
 
 cfg_if! {
-    if #[cfg(all(target_arch = "aarch64", not(target_os = "windows")))] {
+    if #[cfg(all(target_arch = "aarch64", not(any(target_os = "windows", target_os = "macos", target_os = "ios", target_os = "tvos", target_os = "watchos"))))] {
```

Comment:
I guess we can just use vendor = apple here.

Reply:
I tried it but unfortunatly it does not build

```rust
`cfg(target_vendor)` is experimental and subject to change (see issue #29718)
```